console/v1/types_console_link.go

```diff
@@ -19,6 +19,16 @@ type ConsoleLinkSpec struct {
 	Link `json:",inline"`
 	// location determines which location in the console the link will be appended to.
 	Location ConsoleLinkLocation `json:"location"`
+	// applicationMenu holds information about section and icon used for the link in the applicationMenu.
+	ApplicationMenu ApplicationMenuSpec `json:"applicationMenu,omitempty"`
```

Comment:
should this be a pointer since i assume many instances of this won't have one?

Reply:
> should this be a pointer since i assume many instances of this won't have one?

Yes! Thank you for catching that